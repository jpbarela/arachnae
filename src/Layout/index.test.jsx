// @flow
import * as React from "react";
import { render } from "@testing-library/react";
import { Row } from "./index";
import { StyleSheetTestUtils } from "aphrodite";

describe("Row", () => {
  beforeEach(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });
  afterEach(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it("renders", () => {
    const container = render(<Row>I'm a row</Row>);
    expect(container.baseElement.firstChild).toMatchSnapshot();
  });
});
