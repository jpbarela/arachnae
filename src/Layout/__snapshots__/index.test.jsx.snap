// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders 1`] = `
<div>
  <div
    class="row-0-2-1 row-d0-0-2-2 container-0-2-3 container-d0-0-2-4"
  >
    I'm a row
  </div>
</div>
`;
